env/development/global.env

```diff
@@ -44,6 +44,10 @@ MODULE_SITE_BUCKET_PUBLIC_NAME=caendr-dev-site-public
 MODULE_SITE_BUCKET_PRIVATE_NAME=caendr-dev-site-private
 
 MODULE_SITE_SENTRY_NAME="caendr-dev-site-sentry-dsn"
+MODULE_SITE_CART_COOKIE_NAME="caendr-dev-cart-id"
+
+#14 days 
+MODULE_SITE_CART_COOKIE_AGE=1209600
```

Comment:
@orzut good! Wondering if we could add the `unit` of time to the name of the var? e.g. use `MODULE_SITE_CART_COOKIE_AGE_SECONDS` instead of `MODULE_SITE_CART_COOKIE_AGE`